### ISSUE TITLE
Too many jmeterPluginJars == "The filename or extension is too long"

### ISSUE THREAD

**Issue by vlaadbrain** [kulya/jmeter-gradle-plugin#49]
Hello,  i'm experiencing a problem with the use of jmeterPluginJars,  at the moment, because of the way the plugin adds most of the plugins to the command line and i have about 8 jars in there with their full gradle cache path in them.  Including UUID's and whatever else gradle adds to the path.  and that is making the editor java command line too long and thus "The filename or extension is too long" Error. 

i guess i would suggest the old way of adding plugin jars to the classpath. since older plugin versions (way back with version 2.9 of jmeter) used to work with more plugin  jars.

**Comment by foragerr** [comment]
Can you please run gradle with --debug and post the generated classpath value as a gist please? 

aside: Filename or extension too long sounds like a windows limitation rather than a java limitation btw. Are you on windows?

**Comment by vlaadbrain** [comment]
yeah, i'm on windows. i do have a whole lot of extra libs, i don't want to post here though.

**Comment by cfalba80** [comment]
Yes, it is similar, using windows too.
